packages/frontend/src/hooks/useQueryError.ts

```diff
@@ -4,12 +4,21 @@ import { useQueryClient } from '@tanstack/react-query';
 import { useEffect, useState, type Dispatch, type SetStateAction } from 'react';
 import useToaster from './toaster/useToaster';
 
-const useQueryError = (): Dispatch<SetStateAction<ApiError | undefined>> => {
+type opts = {
+    forbiddenToastTitle?: string;
+    forceToastOnForbidden?: boolean;
+};
+
+const useQueryError = ({
+    forbiddenToastTitle,
+    forceToastOnForbidden,
+}: opts = {}): Dispatch<SetStateAction<ApiError | undefined>> => {
     const queryClient = useQueryClient();
     const [errorResponse, setErrorResponse] = useState<ApiError | undefined>();
     const { showToastError } = useToaster();
     useEffect(() => {
         (async function doIfError() {
+            console.log({ errorResponse });
```

Comment:
leftover console log ?

Reply:
whoops, I really need to add your husky script 😮‍💨